### ISSUE TITLE
Add maps data file

### ISSUE THREAD

**Issue by thaddeus** [TarkovTracker/tarkovdata#24]
Add a maps data file with localization data for map names

**Comment by thaddeus** [comment]
Essentially looking for a JSON array like:
```
[
	{
		"id": 0,
		"locale": {
			"en": "Factory"
		},
		"wiki": "https://escapefromtarkov.gamepedia.com/Factory",
	},
	{
		"id": 1,
		"locale": {
			"en": "Customs"
		},
		"wiki": "https://escapefromtarkov.gamepedia.com/Customs"
	}
]
```
Might just be those properties for now, or you can include other metadata if you think of anything that could be valuable for an external tool like TarkovTracker to know about it.

**Comment by thaddeus** [comment]
Issue wasn't closed automatically with the merge of #41 - Thanks for the help @Filodoxia!
